src/views/token-swap/components/quote-list/quote-list.view.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { BigNumber } from 'ethers'
+
+import Quote from '../quote/quote.view'
+import useQuoteListStyles from './quote-list.styles'
+
+function QuoteList ({
+  toToken,
+  quotes,
+  bestQuote,
+  selectedQuote,
+  onQuoteSelect,
+  onShowMoreQuoteInfo
+}) {
+  const classes = useQuoteListStyles()
+
+  function getBestQuoteAmountDiff (quote) {
+    return BigNumber.from(quote.amountToToken).sub(BigNumber.from(bestQuote.amountToToken))
+  }
+
+  return (
+    <div className={classes.root}>
+      {quotes.map((quote) => (
+        <div
+          key={quote.lpId}
```

Comment:
Probably worth mentioning somewhere what lp means (liquidity provider?). It's used in several places and will be confusing for someone new

Reply:
Well it's the unique identifier from the API (yep, it means liquidity provider). But I don't think that it's necessary to explain it somewhere because at the end we are doing the same with a lot of data coming from the Hermez API (for example using `itemId` por the pagination, which is not clear enough without taking a look at the docs)...